lib/capybara/cuprite/driver.rb

```diff
@@ -82,6 +83,10 @@ def find(method, selector)
         browser.find(method, selector).map { |native| Node.new(self, native) }
       end
 
+      def active_element
+        evaluate_script("document.activeElement")
+      end
```

Comment:
There is a [DOM.getDocument](https://chromedevtools.github.io/devtools-protocol/tot/DOM/#method-getDocument) command, but I wasn't able to figure out how to invoke it.

I think it'd be an improvement to define this method on `Browser` instead, then delegate `Driver#send_keys` to `Browser#active_element`.